deployment/kubernetes/yaml/customerservice.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: ftgo-customer-service
+  namespace: default
+spec:
+  ports:
+  - port: 8082
+    targetPort: 8080
+  selector:
+    svc: ftgo-customer-service
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ftgo-customer-service
+  labels:
+    application: ftgo

```

Comment:
The application should be `eventuate-tram-examples-customers-and-orders` - not `ftgo`

Reply:
updated